packages/deployer/src/compiler.ts

```diff
@@ -32,6 +32,7 @@ import {
 import { utils } from './utils/utils';
 
 const ALL_CONTRACTS_IDENTIFIER = '*';
+const SOLC_BIN_DIR = path.join(__dirname, '../../solc_bin');
```

Comment:
The point of using `path.join` is so that the delimiter used is OS specific. Please split `../../solc_bin` into, `'..', '..', 'solc_bin'` so that `path.join` results in the proper delimiter (i.e `\` on windows) for the entire path. Will also open an Airtable issue to do this everywhere, we've been making this mistake in many places.